### PR TITLE
Add PAGE_SIZE constant and replace every occurence of 14 with PAGE_SIZE

### LINKED ISSUE #1045: Remove Offset from paginated API's
Remove the usage of `Offset` from paginated API's

### PR DESCRIPTION
**Pull request by shaileshaanand**
Add PAGE_SIZE constant and replace every occurence of 14 with PAGE_SIZE
closes #1045

Co-authored-by: Divya Garg <divyagarg2601@gmail.com>

### CODE REVIEW

**Review by smartclash — APPROVED** [review]

File: src/Components/Shifting/ListFilter.tsx [hunk 1]
```diff
@@ -117,10 +118,11 @@ export default function ListFilter(props: any) {
 
   useEffect(() => {
     async function fetchData() {
+      console.log("fetching assigned user");
```
Comment (ghost) [inline src/Components/Shifting/ListFilter.tsx:121]: remove console log

File: src/Components/Shifting/ListFilter.tsx [hunk 2]
```diff
@@ -117,10 +118,11 @@ export default function ListFilter(props: any) {
...
       if (filter.assigned_user) {
         setAssignedUserLoading(true);
         const params = {
-          limit: 10,
+          limit: PAGE_LIMIT,
```
Comment (ghost) [inline src/Components/Shifting/ListFilter.tsx:125]: According to PR title, only 14 should be replaced by `PAGE_LIMIT`.  😂 

Hmm there were so inconsistency in page limits.

**Review by abhinandanarya06 — APPROVED** [review]
Do the changes below

Rest LGTM

**Comment by nihal467** [comment]
@gigincg all the pages in every section is having pagination, can we close this PR

**Comment by gigincg** [comment]
Stale/Irrelavant PR

**Pull request closed without merging**